b3lb/rest/models.py

```diff
@@ -400,6 +400,12 @@ class Meta(object):
     def __str__(self):
         return "{}-{}".format(self.tenant.slug, str(self.sub_id).zfill(3))
 
+    @property
+    def get_valid_secret(self) -> str:
```

Comment:
Prefixing the name of a property with `get_` irritates me.

Reply:
obsolete, see secret fragility